tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -412,7 +413,15 @@ public function adminCreatesUserUsingTheProvisioningApi(?string $user):void {
 	 * @throws Exception
 	 */
 	public function userHasBeenCreatedOnDatabaseBackend(?string $user):void {
-		$this->adminCreatesUserUsingTheProvisioningApi($user);
+		$this->createUser(
+			$user,
+			null,
+			null,
+			null,
+			true,
+			'api'
```

Comment:
This should be graph? as this is `ocis` tests?
```suggestion
			'graph'
```

Reply:
this change have been done in this pr https://github.com/owncloud/ocis/pull/7019